docs/guide/setup/defining_protocols.rst

```diff
@@ -0,0 +1,65 @@
+Defining Protocols
+==================
+
+With a thermodynamic cycle of interest identified and represented using `ChemicalSystem` objects,
+the next step is to choose a method for estimating the free energy differences.
+This is done using a :class:`.Protocol`,
+a computational method for estimating the free energy difference between two chemical systems.
+
+Just as there are multiple possible methods for estimating free energy differences,
+there are multiple available Protocols to choose from.
+Included in the ``openfe`` package are the
+:class:`.RelativeHybridTopologyProtocol` and
+the :class:`.AbsoluteSolvationProtocol`.
+This selection is being built upon,
+both by the openfe development team as well as with external academic groups.
```

Comment:
Rather than having a list here that will be difficult to find & update, I would suggest instead having a `Protocols` section to the userguide, where there's a list of Protocols with links to their API docs, relevant userguide entries and cookbooks.

Reply:
Also this skips the PlainMDProtocol

Reply:
I've added stub pages for individual Protocols and referenced #743 as the issue that should populate them